README.md

```diff
@@ -89,7 +116,7 @@ The data is also available in Apache Parquet format. The available fields are de
 
 ### R Package
```

Comment:
Now that there's a longer description of how to download the data, it seems like the R Package section might below below "Download via URL"

Reply:
good idea!

Reply:
I ended up separating out and moving the data formats section above the data access section.